pkg/kubeapiserver/admission/util/initializer.go

```diff
@@ -49,8 +50,29 @@ func IsUpdatingUninitializedObject(a admission.Attributes) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if accessor.GetInitializers() == nil {
+	if initialization.IsInitialized(accessor.GetInitializers()) {
 		return false, nil
 	}
 	return true, nil
 }
+
+// IsInitializationCompletion returns if the operation removes all pending initializers.
```

Comment:
returns true if

Reply:
The original writing is also valid?

Reply:
as written, it sounds like it never returns otherwise.